### DISCUSSION TITLE
new patch version for webpack and prettier not detected

### DISCUSSION THREAD

**Discussion by tomru** [renovatebot/renovate#12915] [Request Help]
Hello there,

I'm unsure why new patch versions of specific npm packages released 7 days ago won't result in a PR being created, specifically 

I've created a minimal test repository https://github.com/tomru/test-webpack-prettier-update 

```sh
❯ npm outdated
Package   Current  Wanted  Latest  Location               Depended by
prettier    2.4.1   2.5.0   2.5.0  node_modules/prettier  test-webpack-prettier-update
webpack    5.64.2  5.64.4  5.64.4  node_modules/webpack   test-webpack-prettier-update
```

I'd expect PRs for prettier@2.5.0 and webpack@5.64.4 but nothing in the logs shows these versions even exist.

I'm not sure if this is due to something in the default configuration ?

<details>
   <summary>Renovate bot debug Logs</summary>
<pre>
DEBUG: No dangling containers to remove
INFO: Repository started
{
  "renovateVersion": "29.29.0"
}
DEBUG: Using localDir: /mnt/renovate/gh/tomru/test-webpack-prettier-update
DEBUG: Repository cache is valid
DEBUG: initRepo("tomru/test-webpack-prettier-update")
DEBUG: Overriding default GitHub endpoint
{
  "endpoint": "https://api.github.com/"
}
DEBUG: tomru/test-webpack-prettier-update default branch = main
DEBUG: Using app token for git init
DEBUG: resetMemCache()
DEBUG: Resetting npmrc
DEBUG: detectSemanticCommits()
DEBUG: checkOnboarding()
DEBUG: isOnboarded()
DEBUG: Checking cached config file name
DEBUG: Existing config file confirmed
DEBUG: Repo is onboarded
DEBUG: migrateAndValidate()
DEBUG: No config migration necessary
DEBUG: massaged config
{
  "config": {
    "extends": [
      "github>whitesource/merge-confidence:beta",
      "config:base"
    ]
  }
}
DEBUG: migrated config
{
  "config": {
    "extends": [
      "github>whitesource/merge-confidence:beta",
      "config:base"
    ]
  }
}
DEBUG: Setting hostRules from config
DEBUG: Found repo ignorePaths
{
  "ignorePaths": [
    "**/node_modules/**",
    "**/bower_components/**",
    "**/vendor/**",
    "**/examples/**",
    "**/__tests__/**",
    "**/test/**",
    "**/tests/**",
    "**/__fixtures__/**"
  ]
}
DEBUG: No vulnerability alerts found
DEBUG: No vulnerability alerts found
DEBUG: findIssue(Dependency Dashboard)
DEBUG: Retrieving issueList
DEBUG: Retrieved 1 issues
DEBUG: Found issue 2
DEBUG: No baseBranches
DEBUG: extract()
DEBUG: Found cached extract
{
  "baseBranch": "main",
  "baseBranchSha": "6aca67b7393d7fb2d83cb35174f2e7fcab10984d"
}
DEBUG: Deleted cached dep updates
INFO: Dependency extraction complete
{
  "baseBranch": "main",
  "stats": {
    "managers": {
      "npm": {
        "fileCount": 1,
        "depCount": 2
      }
    },
    "total": {
      "fileCount": 1,
      "depCount": 2
    }
  }
}
DEBUG: Package releases lookups complete
{
  "baseBranch": "main"
}
DEBUG: branchifyUpgrades
DEBUG: 0 flattened updates found:
DEBUG: Returning 0 branch(es)
DEBUG: config.repoIsOnboarded=true
DEBUG: packageFiles with updates
{
  "config": {
    "npm": [
      {
        "packageFile": "package.json",
        "deps": [
          {
            "depType": "dependencies",
            "depName": "prettier",
            "currentValue": "^2.4.1",
            "datasource": "npm",
            "prettyDepType": "dependency",
            "lockedVersion": "2.4.1",
            "depIndex": 0,
            "warnings": [],
            "versioning": "npm",
            "sourceUrl": "https://github.com/prettier/prettier",
            "homepage": "https://prettier.io",
            "currentVersion": "2.4.1",
            "fixedVersion": "2.4.1",
            "updates": []
          },
          {
            "depType": "dependencies",
            "depName": "webpack",
            "currentValue": "^5.64.2",
            "datasource": "npm",
            "prettyDepType": "dependency",
            "lockedVersion": "5.64.2",
            "depIndex": 1,
            "warnings": [],
            "versioning": "npm",
            "sourceUrl": "https://github.com/webpack/webpack",
            "currentVersion": "5.64.2",
            "fixedVersion": "5.64.2",
            "updates": []
          }
        ],
        "packageJsonName": "test-webpack-prettier-update",
        "packageFileVersion": "1.0.0",
        "packageJsonType": "library",
        "npmLock": "package-lock.json",
        "managerData": {
          "yarnZeroInstall": false
        },
        "skipInstalls": true,
        "constraints": {},
        "lockFiles": [
          "package-lock.json"
        ]
      }
    ]
  }
}
DEBUG: processRepo()
DEBUG: Processing 0 branches:
DEBUG: Calculating hourly PRs remaining
DEBUG: Retrieving PR list
DEBUG: Retrieved 1 Pull Requests
DEBUG: currentHourStart=2021-12-02T09:00:00.000+00:00
DEBUG: PR hourly limit remaining: 2
DEBUG: Calculating prConcurrentLimit (20)
DEBUG: 0 PRs are currently open
DEBUG: PR concurrent limit remaining: 20
DEBUG: Calculated maximum PRs remaining this run
{
  "prsRemaining": 2
}
DEBUG: PullRequests limit = 2
DEBUG: Calculating hourly PRs remaining
DEBUG: currentHourStart=2021-12-02T09:00:00.000+00:00
DEBUG: PR hourly limit remaining: 2
DEBUG: Calculating branchConcurrentLimit (20)
DEBUG: 0 already existing branches found:
DEBUG: Branch concurrent limit remaining: 20
DEBUG: Calculated maximum branches remaining this run
{
  "branchesRemaining": 2
}
DEBUG: Branches limit = 2
DEBUG: Ensuring Dependency Dashboard
DEBUG: ensureIssue(Dependency Dashboard)
DEBUG: Issue is open and up to date - nothing to do
DEBUG: Removing any stale branches
DEBUG: config.repoIsOnboarded=true
DEBUG: No renovate branches found
DEBUG: Repository timing splits (milliseconds)
{
  "splits": {
    "init": 2134,
    "extract": 454,
    "lookup": 377,
    "update": 277
  },
  "total": 3443
}
DEBUG: http statistics
{
  "urls": {
    "https://api.github.com/graphql (POST)": 3,
    "https://api.github.com/repos/tomru/test-webpack-prettier-update/contents/renovate.json (GET)": 1,
    "https://api.github.com/repos/tomru/test-webpack-prettier-update/issues/2 (GET)": 2,
    "https://api.github.com/repos/tomru/test-webpack-prettier-update/pulls (GET)": 1,
    "https://api.github.com/repos/whitesource/merge-confidence/contents/beta.json (GET)": 1
  },
  "hostStats": {
    "api.github.com": {
      "requestCount": 8,
      "requestAvgMs": 216,
      "queueAvgMs": 0
    }
  },
  "totalRequests": 8
}
INFO: Repository finished
{
  "durationMs": 3443
}
</pre>
</details>

**Comment by rarkins** [comment]
Look up "rangeStrategy" in the docs, I think you'll want rangeStrategy=update-lockfile

**Reply by tomru** [reply]
ah, I wasn't aware of the default. Thanks!
